Python/Product/TestAdapter.Executor/Pytest/PytestExtensions.cs

```diff
@@ -35,47 +36,64 @@ static class PyTestExtensions {
             var sourceParts = sourceAndLineNum.Split(':');
             if (sourceParts.Length != 2 ||
                 !Int32.TryParse(sourceParts[1], out line)) {
-                throw new FormatException(String.Format("Invalid source: {0}",test.ToString()));
+                throw new FormatException(String.Format("Invalid source: {0}", test.ToString()));
```

Comment:
Should be localized, and use `.FormatUI`, since we sometimes print exception message to the output window. Also no need to call `ToString()`.

Reply:
will do